app/components/recirculation-prefooter.js

```diff
@@ -86,27 +80,18 @@ export default Component.extend(
     actions: {
       postClick(post, index) {
         const labels = ['footer', `footer-slot-${index + 1}`];
-        let additionalRecommendationData;
 
-        if (this.shouldUseExperimentalRecommendationService) {
```

Comment:
Seems like mobile wiki was never changed to permanently use the recommendations service, like in /app https://github.com/Wikia/app/pull/16934/files
Gonna have to change it in another ticket